ledger/streaming/client/benches/bench.rs

```diff
@@ -0,0 +1,28 @@
+#![feature(test)]
+
+extern crate test;
+
+use futures::{executor::block_on, StreamExt};
+use mc_ledger_db::test_utils::MockLedger;
+use mc_ledger_streaming_api::{test_utils::get_custom_block_stream, BlockStream};
+use mc_ledger_streaming_client::block_validator::BlockValidator;
+use test::Bencher;
+
+/// Process 1000 typically sized blocks
+#[bench]
+fn process_1000_blocks(b: &mut Bencher) {
+    let logger = mc_common::logger::create_test_logger("benchmark_logger".to_string());
+    let input_stream_builder = get_custom_block_stream(1, 3, 1000, 2);
+    let ledger: Option<MockLedger> = None;
+    let output_stream_builder = BlockValidator::new(input_stream_builder, ledger, logger);
```

Comment:
Nit: This declaration is overly verbose.
```suggestion
    let output_stream_builder = BlockValidator::new(input_stream_builder, None, logger);
```

Reply:
It's required for the automatic type inference due to the use of generics. If it's specified as none, it can't compile time infer it.